assets/js/components/dashboard-sharing/ModuleRecoveryAlert.js

```diff
@@ -218,9 +213,18 @@ export default function ModuleRecoveryAlert() {
 				'Dashboard data for some services has been interrupted',
 				'google-site-kit'
 			) }
+			ctaLabel={ ctaLabel }
+			onCTAClick={ onCTAClick }
+			ctaLink={ ctaLink }
 			description={ description }
 			learnMoreURL="https://sitekit.withgoogle.com/documentation/using-site-kit/dashboard-sharing/"
 			learnMoreLabel={ __( 'Learn more', 'google-site-kit' ) }
+			dismiss={
+				isLoading
+					? undefined
+					: __( 'Remind me later', 'google-site-kit' )
+			}
```

Comment:
Is this condition necessary here? If `isLoading` is true then the `BannerNotification` wouldn't be shown at all but would show the `ProgressBar` instead, correct?

Reply:
The `<ProgressBar>` is passed into the `children` prop of `BannerNotification` instead of the whole `BannerNotification` itself. This is to allow the `title` of the notification to be rendered and only show the rest of the component as loading. So the `dismiss` link still shows if passed as its own prop below the progress bar.

Reply:
Ah in that case, I think it would be more appropriate to pass  `isDismissible` as `false` which we could set in the same condition where `children` is set – is there a reason we can't do that instead?